test/functional/mempool_reorg.py

```diff
@@ -92,7 +92,7 @@ def test_reorg_relay(self):
             # However, the node will answer requests for the tx from the recently-disconnected block.
             assert_equal(peer1.last_message["tx"].tx.getwtxid(),tx_disconnected["tx"].getwtxid())
 
-        self.nodes[1].setmocktime(int(time.time()) + 30)
+        self.nodes[1].setmocktime(int(time.time()) + 300)
```

Comment:
Seems like due to poisson announcement timer, with a small probability, the announcement hasn't been scheduled yet. A more sure way to fix this is to revert to system time and just wait for the invs:

```suggestion
        # Reset to system time
        self.nodes[1].setmocktime(0)
        peer1.wait_for_broadcast([tx_before_reorg["tx"].getwtxid(), tx_child["tx"].getwtxid(), tx_disconnected["tx"].getwtxid()])
```

(And then delete L97-101)

Reply:
Oh true. I suppose in the very rare scenario that it's a really long time, fastforward + immediately fail is better than waiting 60 seconds and failing. Would prefer 60 over 300 but nbd.